doc/Building.md

```diff
@@ -65,6 +65,13 @@ sudo apt-get install autoconf automake build-essential ca-certificates cmake \
     lua5.1 luarocks meson nasm ninja-build patch perl pkg-config unzip wget
 ```
 
+**Note:** Debian distros might need `meson` to be installed from bookworm-backport) becuase the one included in repos is quite old:
```

Comment:
```suggestion
**Note:** Debian distributions might need `meson` to be installed from bookworm-backport) because the one provided by the default repositories is quite old:
```